src/main/app/case/index.ts

```diff
@@ -1,14 +1,17 @@
+import * as path from 'path';
 import * as express from 'express';
 
 import { AuthorizationMiddleware } from 'idam/authorizationMiddleware';
+import { RouterFinder } from 'router/routerFinder';
 
 export class AdoptionApplication {
   enableFor (app: express.Express): void {
     app.all('/case/*', this.requestHandler());
+    app.use('/', RouterFinder.findAll(path.join(__dirname, 'routes')));
   }
 
   requestHandler() {
-    const requiredRoles = [];
+    const requiredRoles = ['citizen'];
```

Comment:
Is it better if we get roles from the config?

Reply:
yes, i will do that..

Reply:
done